### PR TITLE
resolve issue with sort icons on admin/users and admin/runners

### LINKED ISSUE #24327: The sort icon problem in admin setting pages
### Description

Only in this page the sort icon is below the column title
image

The others are correct:
image
image

### Gitea Version

latest

### Can you reproduce the bug on the Gitea demo site?

No

### Log Gist

_No response_

### Screenshots

_No response_

### Git Version

_No response_

### Operating System

_No response_

### How are you running Gitea?

build

### Database

None

### PR DESCRIPTION
**Pull request by jladbrook**
Fixes #24327 to avoid the sort icon changing the table header over multiple lines and adds missing sort icons on the runners page.

## Fixed User Admin Page

image

## Updated Runners Page

image

### CODE REVIEW

File: web_src/css/runner.css [hunk 1]
```diff
@@ -52,3 +52,10 @@
   background-color: var(--color-yellow);
   color: var(--color-white);
 }
+
+.runner-container .runner-tags {
+    display: flex;
```
Comment (wxiaoguang) [inline web_src/css/runner.css]: It doesn't look good to me to change a `td`'s "display" like this.

Comment (jladbrook) [reply web_src/css/runner.css]: yes you're right; it wasn't really adding anything to the implementation so it has been removed in the latest commit

**Comment by silverwind** [comment]
It seems https://github.com/go-gitea/gitea/pull/24370 had adressed some of the same issues as this PR. @wxiaoguang which parts of this PR do we still need?

**Comment by silverwind** [comment]
Looking at this admin user table, I can't help but feel that there are too many columns to fit in the available space, so either remove some columns or increase page width, imho. Maybe the checkbox-only column titles could be reduced to svg icons only to save space.

**Comment by wxiaoguang** [comment]
> which parts of this PR do we still need?

If I understand correctly, this PR is mainly for the "sort" icons/logic?

**Comment by silverwind** [comment]
I get the same impression, so the CSS changes should potentially be backed out now.

**Comment by denyskon** [comment]
@jladbrook Looks like the first issue is resolved now. Could you refresh your PR and only keep the fix for the runners page?

**Comment by jladbrook** [comment]
@denyskon that's done

**Review by denyskon — APPROVED** [review]

**Comment by wxiaoguang** [comment]
Has the alignment problem  https://github.com/go-gitea/gitea/pull/24360#issuecomment-1528095766 been resolved? 

If yes, please update the screenshot.

**Comment by jladbrook** [comment]
> Has the alignment problem [#24360 (comment)](https://github.com/go-gitea/gitea/pull/24360#issuecomment-1528095766) been resolved?
> 
> If yes, please update the screenshot.
> 
> If no and need help, feel free to ask ~~
> 
> Update: IMO it doesn't need to be considered as "bug" or need to be backported.

Hi @wxiaoguang that wasn't included in the PR previously. I've had a quick look and updated to include a change to achieve:

image

This was a minor change to base.css - if there is a better way to achieve please let me know and I'll update accordingly.

**Review by wxiaoguang — APPROVED** [review]
Thank you! It looks good to me now (and I will take a test later 😄 )

----

Update: the new `margin` affects other UI. I will propose a fix soon.

**Comment by jladbrook** [comment]
> Update: the new `margin` affects other UI. I will propose a fix soon.

Oh sorry I did check around in my changed version and didn't notice the change - what page(s) was affected?

**Comment by wxiaoguang** [comment]
That's really strange. According to my test, without `margin-top: 0.25rem;` , the old UI just looks fine.

But when the `margin-top: 0.25rem;` is added, it makes the UI misalign.

I don't know why your first screenshot shows that the icon misalignment. Maybe it's better to remove the `margin-top: 0.25rem;` and leave the problem to the future.

**Comment by denyskon** [comment]
@wxiaoguang Why are you against a backport? This fix adds the missing sort icon to the runners table and improves sorting. I think this isn't anything that has the potential to break something

**Comment by wxiaoguang** [comment]
IMO each backport comes with risk: even if there is no conflict, the backported code might not work, and no one would really test the backport.

So, if a fixed problem is not very important (eg: no matter with or without an "icon", the "list" still works), I would prefer to not backport.

But that's just my opinion, free free to backport if you think it is important.

**Pull request merged**